app_dart/lib/src/service/branch_service.dart

```diff
@@ -113,4 +113,18 @@ class BranchService {
 
     throw InternalServerError('Failed to find a revision to branch Flutter recipes for $branch');
   }
+
+  Future<List<String>> getStableBetaDevBranches({required gh.GitHub github, required gh.RepositorySlug slug}) async {
+    String devSha = (await github.repositories.getBranch(slug, 'dev')).commit!.sha!;
```

Comment:
Constants?

Reply:
Sure, added final keywords for all the strings.